README.md

```diff
@@ -56,30 +67,28 @@ Also make sure to include the following in order to import the compiled CSS from
 
 **[More info about using USWDS CSS & SCSS](./docs/scss.md)**
 
-## Roadmap
-
-- [x] Add lint configs and pre-commit hooks for contributing
-- [x] Set up CI for running tests and linting
-- [x] Add support for and convert existing component(s) to TypeScript
-- [x] Decide on and document git workflow for this project
-- [x] Decide on and document release workflow for the package
-- [x] Load and export USWDS CSS
-- [x] Load and export USWDS fonts/svgs/other assets
-- [x] Decide on long-term lib publishing/hosting solution
-- [x] Add CI status badge
-- [x] Setup https://danger.systems/js/ to check on contribution standards, possibly even checking yarn install for warnings
-- [ ] Document decision behind node version and upgrade plan
-- [ ] ADR to decide on and set up a React component test helper:
-  - https://airbnb.io/enzyme/
-  - https://testing-library.com/docs/react-testing-library/
-- [ ] Enable `pkg.module` entrypoint for better module and tree shaking:
-  - https://github.com/rollup/rollup/wiki/pkg.module
-  - https://stackoverflow.com/questions/41289200/output-an-es-module-using-webpack
-- [ ] Add more documentation around how to contribute and write new components
-- [ ] Add component scaffolding shortcut (for generating component, tests, stories files with template code)
-- [ ] Add testing coverage collection
-- [ ] Set up Storybook as public Github page
-- [ ] Add example application that uses the library to the repo
-- [ ] Add visual testing automation tool (i.e., Loki)
-- [ ] Enforce adding to CHANGELOG when merging a PR into develop
-- [ ] Make sure new components are added as package exports
+## Maintainers
+
+- [@suzubara](https://github.com/suzubara)
```

Comment:
***Please add yourself here as part of this PR!*** (if you read the doc about active maintainers and would like to be one right off the bat)